src/ScriptCs.Core/ScriptExecutor.cs

```diff
@@ -10,8 +12,11 @@ namespace ScriptCs
 {
     public class ScriptExecutor : IScriptExecutor
     {
-        public static readonly string[] DefaultReferences = new[] { "System", "System.Core", "System.Data", "System.Data.DataSetExtensions", "System.Xml", "System.Xml.Linq", "System.Net.Http" };
+        public static readonly string[] DefaultReferences = new[] { "System", "System.Core", "System.Data", "System.Data.DataSetExtensions", "System.Xml", "System.Xml.Linq", "System.Net.Http", typeof(ScriptExecutor).Assembly.Location };
```

Comment:
Because the ScriptLibraryWrapper is in ScriptCs.Core and that is what the wrapper class that is generated derives from. We already added this for Roslyn and were not doing it for Mono. Now it has just been upleveled.